File: app/src/main/java/com/eericxu/wallpaper/utils/MathT.java
package com.eericxu.wallpaper.utils;

/**
 * Created by Eericxu on 2017-09-04.
 */

public class MathT {
	public static double sin(int angle) {
		return Math.sin(angleToNum(angle));
	}

	public static double cos(int angle) {
		return Math.cos(angleToNum(angle));
	}

	public static double angleToNum(int angle) {
		return 2 * Math.PI / 360 * angle;
	}
}

File: app/src/main/java/com/eericxu/wallpaper/utils/MathT.kt
package com.eericxu.wallpaper.utils

/**
 * Created by Eericxu on 2017-09-04.
 */
object MathT {
    fun sin(angle: Int): Double {
        return kotlin.math.sin(angleToNum(angle))
    }

    fun cos(angle: Int): Double {
        return kotlin.math.cos(angleToNum(angle))
    }

    fun angleToNum(angle: Int): Double {
        return 2 * Math.PI / 360 * angle
    }
}
